can-define.js

```diff
@@ -288,6 +288,7 @@ make = {
 		events: function(prop, getCurrent, setData, eventType) {
 			return function(newVal) {
 				if (this.__inSetup) {
+					Observation.ignore(getCurrent).call(this);
```

Comment:
I had noticed something similar when adding the changes. Justin had requests 3 changes, but I noticed that any 1 of them would make it pass.